src/App.vue

```diff
@@ -46,21 +43,19 @@ export default {
 </script>
 
 <style lang="scss">
+    @import "_scss/default";
+
     .container{
```

Comment:
i think we agree to remove `.container` from the App.vue because it can ruin the login page and define the container on each page instead